src/Workspaces/CSharp/Portable/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -1201,5 +1201,15 @@ public static ConditionalAccessExpressionSyntax GetParentConditionalAccessExpres
 
             return null;
         }
+
+        public static ConditionalAccessExpressionSyntax GetInnerMostConditionalAccessExpression(this SyntaxNode node)
+        {
+            if (!(node is ConditionalAccessExpressionSyntax))
+            {
+                return null;
+            }
+
+            return node.DescendantNodesAndSelf().OfType<ConditionalAccessExpressionSyntax>().Last();
```

Comment:
Probably not what you intended.  Consider lambdas.

```
a?.M(x => x?.P);
```

Calling this on the `ConditionalAccessExpression` for `a?.M` will return `x?.P`.